Nos/Views/ProfileKnownFollowersView.swift

```diff
@@ -17,28 +17,16 @@ struct ProfileKnownFollowersView: View {
             if let second = knownFollowers[safe: 1]?.source {
                 StackedAvatarsView(avatarUrls: [first.profilePhotoURL, second.profilePhotoURL])
                 if followers.count > 2 {
-                    Text(Localized.followedByTwoAndMore.localizedMarkdown([
-                        "one": first.safeName,
-                        "two": second.safeName,
-                        "count": "\(followers.count - 2)"
-                    ]))
+                    Text(.localizable.followedByTwoAndMore(first.safeName, second.safeName, followers.count - 2))
```

Comment:
I wasn't able to figure out if named arguments could be supported. Leaving that as an open question.